src/gluonts/evaluation/_base.py

```diff
@@ -90,6 +92,14 @@ class Evaluator:
         which is computationally expensive to evaluate and thus slows
         down the evaluation process considerably.
         By default False.
+    num_workers
+        The number of multiprocessing workers that will be used to process
+        the data in parallel.
+        Default is multiprocessing.cpu_count().
+        Setting it to 0 means no multiprocessing.
+    chunk_size
+        Controls the approximate chunk size each workers handles at a time.
+        Default is 32.
```

Comment:
Optimal `chunck_size` will depend on the operating system and the page sizes etc. and how much the cpu is prefetching.

Reply:
32 seems to be good, but basically 8 to 64 was about as good too.